README.md

```diff
@@ -18,135 +19,87 @@ controller.
-```
+This package is base on ament_cmake hence one can simply use the standard:
 
-If you need another terminal to connect to the last docker container ran you can use:
-```
-docker exec --workdir=/ws -u user -it `docker ps --latest --quiet` bash
-```
-
-Once connected to docker with multiple terminal (5) please run in order:
-
-- Terminal 1: Start the simulation:
-```
-reset && catkin build linear_feedback_controller && source install/setup.bash && roslaunch talos_pal_physics_simulator talos_pal_physics_simulator_with_actuators.launch robot:=full_v2
-```
-
-- Terminal 2: Spawn the default controller:
-```
-reset && source install/setup.bash && roslaunch talos_controller_configuration default_controllers.launch
-```
-in the same terminal, once the robot is in the default pose, kill (`ctrl+C`) the roslaunch
-
-- Terminal 3: Spawn the linear feedback controller:
-```
-reset && source install/setup.bash && roslaunch linear_feedback_controller talos_linear_feedback_controller.launch simulation:=true default_params:=true
-```
-
-- Terminal 4: For recording logs you can use `rosbag`:
-```
-reset && source install/setup.bash && rosbag record -o src/lfc /linear_feedback_controller/sensor_state /linear_feedback_controller/desired_control
-```
-
-- Terminal 5: For starting a very simple PD controller via Feedback gains run:
-```
-reset && source install/setup.bash && rosrun linear_feedback_controller pd_controller
-```
-
-The next paragraph is the same procedure except one can use the Makefile that is
-in the root of this repos to do so.
-
-### Build the repository using make
-
-There is also [Makefile](Makefile) to ease the usage of docker and ROS instructions
-in order to execute the demo.
-
-- Create the workspace and clone the repository
-```
-mkdir -p ~/devel/workspace/src
-cd ~/devel/workspace/src
+```bash
 git clone --recursive git@github.com:loco-3d/linear-feedback-controller.git
+cd linear-feedback-controller
+make _build
+cd _build
+cmake .. -DCMAKE_INSTALL_PREFIX=/path/to/your/install/folder
+make
+make install
 ```
 
-- Run the docker and build in terminal(1)
-```
-cd ~/devel/workspace/src/linear_feedback_controller
-make docker_pull_and_run
-make build
-```
-
-- Run the simulation, terminal(1)
-```
-make simu
-```
+Or one can use the ROS2 super build system [colcon](https://colcon.readthedocs.io/en/released/):
 
-- Run the default controller and kill it (ctrl+C) when the robot is in half-sitting terminal(2)
-```
-make docker_connect
-make default_ctrl
+```bash
+mkdir -p workspace/src
+cd workspace/src
+git clone --recursive git@github.com:loco-3d/linear-feedback-controller.git
+cd ../../ # back to workspace.
+colcon build
 ```
 
-- Run the linear feedback controller terminal (2)
-```
-make lf_ctrl
+One can also use [nix](https://nixos.wiki/wiki/Main_Page) to:
+- Check the package (builds and run tests):
+```bash
+git clone --recursive git@github.com:loco-3d/linear-feedback-controller.git
```

Comment:
```suggestion
git clone --recursive https://github.com/loco-3d/linear-feedback-controller.git
```

The HTTPS protocol will work out of the box for anybody, while the SSH protocol require more things (github account + ssh key).

And in the cases one has the ssh key configured with your github account, they can set their gitconfig with 
```
[url "git@github.com:"]
	insteadOf = "https://github.com/"
```